pyproject.toml

```diff
@@ -1,18 +1,18 @@
 [tool.poetry]
 name = "authutils"
-version = "5.0.4"
+version = "5.0.5"
 description = "Gen3 auth utility functions"
 authors = ["CTDS UChicago <cdis@uchicago.edu>"]
 license = "Apache-2.0"
 
 [tool.poetry.dependencies]
 python = "^3.6"
 cached-property = "~=1.4"
-cdiserrors = "~=1.0"
+cdiserrors = "<2.0.0"
 xmltodict = "~=0.9"
 
 authlib = "~=0.11"
-httpx = "^0.12.1"
+httpx = "^0.14.0"
```

Comment:
why?

Reply:
can't poetry resolve that?

Reply:
```
gen3authz (1.0.3) depends on httpx (>=0.14.0,<0.15.0), gen3authz (1.0.3) is incompatible with authutils (>=5.0.0,<6.0.0).
```

Reply:
replacing with `>=0.12.1,<1.0.0`
for context: https://python-poetry.org/docs/dependency-specification/
`^0.2.3	>=0.2.3 <0.3.0`